### ISSUE TITLE
Spektrum DSM surface

### ISSUE THREAD

**Issue by RobertoD91** [pascallanger/DIY-Multiprotocol-TX-Module#972]
TX: Spektrum DX3s
RX: Spektrum SR300 and Spektrum SR3001

According to this pdf  [Spektrum_Surface_Receiver_Chart.pdf](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/files/15176280/Spektrum_Surface_Receiver_Chart.pdf) these receivers are only compatible with surface DSM and actually do not work with either DSMR or DSM2SFC.

#855 

I will capture the spi signal asap.

**Comment by pascallanger** [comment]
Right, DSM is not supported only DSM2 and DSMR.
Let's see if you have access to the cyrf6936. You might look at the receiver side as well.

**Comment by pascallanger** [comment]
> But searching the fcc id we can see a CYRF6936 inside https://fcc.report/FCC-ID/BRWDAMTX11/1166223 this is not useful ?

Yes there is a cyrf6936 but there is also a controller on the board so the board pins are useless.
You would need to connect wires to the 5 vias within circle + gnd: 
![image](https://github.com/pascallanger/DIY-Multiprotocol-TX-Module/assets/6367598/6bd44477-1d50-453f-b458-17f06d073893)
If you proceed just send dumps like you are doing with Traxxas TQ1.

**Comment by mozgy** [comment]
> > You would need to connect wires to the 5 vias within circle + gnd:
> 
> I don't think that i can solder a cable in this small vias :( Any suggestion?

PCBite is perfect tool here.

**Comment by tmur11** [comment]
@RobertoD91 were you successful in getting a logix dump of these older spectrum DSM recievers?

**Comment by RobertoD91** [comment]
> [@RobertoD91](https://github.com/RobertoD91) were you successful in getting a logix dump of these older spectrum DSM recievers?

Hi, if I remember correctly, I sent one unit to pascallanger.

**Comment by tmur11** [comment]
ok thanks, @pascallanger did you ever get a chance to look at this reciever? Would be willing to donate funds if needed.

**Comment by sangyue** [comment]
DSM protocol is too old...
You can have a try with below aftermarket DSMR receivers develop by DasMikro. It works smoothly with Spektrum latest transmitters in DSMR and SLT protocol. It is a 5 channel version and function is similar to SR515.
[https://dasmikro.com/product/spektrum-compatible-product-dsk-515n-5ch-dsmr-and-slt-dual-protocol-antenna-free-receiver/](url)
